.ci/pylint

```diff
@@ -9,11 +9,9 @@ find src/ -name '*.py[c|~]' -delete
 find src/ -regex "\(.*__pycache__.*\|*.py[co]\)" -delete
 
 # E1129=doesn't like Fabric's context manager
-# E1101=no-member, temporary while python2 is still supported
```

Comment:
python2 is still supported, but if this can be removed, all the better

Reply:
I use python2 locally and all was fine. I pushed the pylint disable comment into the file itself, so it's still there, just not affecting the *entire* project